ksqldb-benchmark/src/main/java/io/confluent/ksql/benchmark/SerdeBenchmark.java

```diff
@@ -183,6 +184,7 @@ private static RowGenerator getRowGenerator(final String schemaName) throws IOEx
       return new RowGenerator(generator, keyField, Optional.empty());
     }
 
+    @SuppressFBWarnings("RCN_REDUNDANT_NULLCHECK_OF_NONNULL_VALUE")
```

Comment:
My local build fails without this. I don't know why ksql Jenkins passes, but even cleaning my MVN repo was failing.